packages/big-design/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,38 @@
+import { ExpandLessIcon, ExpandMoreIcon } from '@bigcommerce/big-design-icons';
+import React, { useState } from 'react';
+
+import { Flex } from '../Flex';
+
+import { StyledTitle } from './styled';
+
+export interface AccordionProps {
+  title: string;
+  onToggle?(): void;
```

Comment:
Should this be something like `onChange?(isOpen: boolean): void`

Reply:
Yes, it's a better version. Changed `onToggle?(): void;` to `onChange?(isOpen: boolean): void`.